rewording in supported_model_formats (#14334)
openvinotoolkit/openvino: docs/MO_DG/prepare_model/convert_model/supported_model_formats.md

**ONNX, PaddlePaddle** - formats supported directly, which means they can be used with OpenVINO Runtime without any prior conversion. For a guide on how to run inference on ONNX and PaddlePaddle, see how to [Integrate OpenVINO™ with Your Application](../../../OV_Runtime_UG/integrate_with_your_application.md).

> **TensorFlow, PyTorch, MXNet, Caffe, Kaldi** - formats supported indirectly, which means you need to convert them before running inference. Conversion from these formats to OpenVINO IR is performed with Model Optimizer and in some cases may involve intermediate steps.

Reviewer: ```suggestion
**TensorFlow, PyTorch, MXNet, Caffe, Kaldi** - requires offline conversion to IR before running inference. Conversion from these formats to OpenVINO IR is performed with Model Optimizer and in some cases may involve intermediate steps.
```

> **TensorFlow, PyTorch, MXNet, Caffe, Kaldi** - formats supported indirectly, which means you need to convert them before running inference. Conversion from these formats to OpenVINO IR is performed with Model Optimizer and in some cases may involve intermediate steps.

Author: ```suggestion
**TensorFlow, PyTorch, MXNet, Caffe, Kaldi** - formats supported indirectly, which means they need to be converted to OpenVINO IR before running inference. The conversion is done with Model Optimizer and in some cases may involve intermediate steps.
```
